### DISCUSSION TITLE
Merging issues

### DISCUSSION THREAD

**Discussion by 0xpr03** [F43nd1r/Acrarium#331] [Q&A]
Let's say there's an issue way before the actual stacktraces, creating different ones. Can I somehow merge issues  (and their reports) ?

**Comment by F43nd1r** [comment]
Yes, just select the bugs you want to merge in the bug view and click the merge button.

**Reply by 0xpr03** [reply]
![image](https://user-images.githubusercontent.com/1280209/159184486-f10d3b1e-8973-4b0e-acfc-af9aee086312.png)
I can't see that option here. (Not different with full width)

**Reply by 0xpr03** [reply]
(and the menus right top are just filter + shown columns)

**Reply by 0xpr03** [reply]
Am I missing something here ? There's no merge button, even though [there should be something..](https://github.com/F43nd1r/Acrarium/blob/df8427a0522179ac2f74806bacf2c1da5820fd74/acrarium/src/main/kotlin/com/faendir/acra/ui/view/app/tabs/BugTab.kt#L71).
